src/FixerDescriptionAwareInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+interface FixerDescriptionAwareInterface
+{
+    /**
+     * @return FixerDescription
+     */
+    public function getFixerDescription();
```

Comment:
introducing the new interface and change the existing one (i.e. (re)move the `getDescription` methid to the new interface as well)